templates/slim/section.html.slim

```diff
@@ -1,8 +1,37 @@
-/ support for vertical slides
+/ OPTIONS PROCESSING
+/ extracting block image attributes to find an image to use as a background attribute
+- data_background, data_background_size, data_background_repeat,
+  data_background_transition = nil
+/ FIXME would have expected the following to work... which would probably be faster
+/- slide_images = find_by(:context => :image) {|image| image.attr(1) == 'background' }
+- slide_images = find_by(:context => :image)
```

Comment:
Instead of iterating on all of the section's images shouldn't I be able to prefilter the results? I tried but I failed. cc @mojavelinux